infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/logreader/StreamsLogEntryReader.java

```diff
@@ -146,6 +158,14 @@ private boolean isValidTransactionEntry(@NonNull OpaqueEntry entry) {
             return false;
         }
 
+        // In DYNAMIC mode, chances are that some streams to replicate were not registered when LogReplicationConfig
```

Comment:
Can you reword the comment to something like -
'... tables corresponding to some streams to replicate were not opened when LogReplicationConfig was initialized.  So these streams will be missing from the list of streams to replicate.  Check the registry table and add them to the list in that case'